client/src/components/modals/modal.js

```diff
@@ -8,13 +8,13 @@ const Modal = (props) => {
     showModal,
     content,
   } = props;
-  // onKeyDown={toggleModal} role="button" tabIndex="0"
+
   return (
     <div className={`modal ${modalClass}`} style={{ display: showModal ? 'block' : 'none' }}>
-      <div className="backdrop" onClick={toggleModal} />
-      <div className="modal-content">
+      <div className="backdrop" onClick={toggleModal} onKeyDown={toggleModal} role="button" tabIndex="0" />
```

Comment:
What does this keydown do? Is it being properly triggered? I remember I had trouble with the keydown triggering

Reply:
I never got the keydown working. I actually forgot to complete the functionality. Will work on getting this setup for closing the modals when I refactor them to a single file.